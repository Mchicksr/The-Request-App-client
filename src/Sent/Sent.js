import React from 'react';
import './Sent.css'
import {  Link } from 'react-router-dom'


function Sent(){

    return(
    <div>
        <h1 className="neon">THANK YOU!</h1><br></br>
        <h1 className="neon">THE DJ HAS YOUR REQUEST!</h1> 
        <Link to="Vote" className = "req">Post Request</Link>
        <Link to="Vote" className ="req">Vote</Link>
    </div>
    )


}

export default Sent;