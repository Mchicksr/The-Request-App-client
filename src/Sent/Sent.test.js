import React from 'react'
import Sent from './Sent'
import { shallow } from 'enzyme'
import { configure } from 'enzyme'
import ReactDOM from 'react-dom'
import Adapter from 'enzyme-adapter-react-16'
configure({adapter:new Adapter()})
import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'

it('renders without crashing,',()=>{
    const div = document.createElement('div');
    ReactDOM.render( <BrowserRouter ><Sent /></BrowserRouter>, div)
    })


it("matches snapshot sent", () => {
    // const tree = renderer.create(<Button label="click me please"></Button>).toJSON();
      const tree = renderer.create(  <a class="req" href="/Sent"> Send Request </a>).toJSON();
  
    expect(tree).toMatchSnapshot();
  })