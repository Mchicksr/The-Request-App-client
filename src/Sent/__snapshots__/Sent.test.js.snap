// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot sent 1`] = `
<a
  class="req"
  href="/Sent"
>
   Send Request 
</a>
`;
